doc/howto/move_instances.md

```diff
@@ -59,6 +59,12 @@ To do so, ensure the following configuration:
 * Set {config:option}`instance-migration:migration.stateful` to `true` on the instance.
```

Comment:
Can you add to the intro of this list that you only need these settings for local storage backends?
It's always bad style to write "Do this now! Or actually, maybe you didn't need to ..." ;)